README.md

```diff
@@ -112,18 +111,18 @@ This:
 * Sets them as `staging` and `production` Git remotes
 * Configures staging with `RACK_ENV` and `RAILS_ENV` environment variables set
   to `staging`
-* Adds the [Rails Stdout Logging][logging-gem] gem
-  to configure the app to log to standard out,
-  which is how [Heroku's logging][heroku-logging] works.
+* Adds the [Rails 12factor][rails-12factor] gem
+* Adds free Sendgrid add-on and configure SMTP settings to use it
```

Comment:
I've been using mandril on projects lately and found it to be a better product 
compare:
https://addons.heroku.com/sendgrid
https://addons.heroku.com/mandrill